src/routes/assets/+page.svelte

```diff
@@ -1,648 +1,666 @@
 <script lang="ts">
-	import { DEFAULT_DESCRIPTION, DEFAULT_HOST } from '$lib/utils/metadata';
-	import { Main } from '$lib/layouts';
-	import { TITLE_SUFFIX } from '$routes/titles';
-	import FooterNav from '../../lib/components/FooterNav.svelte';
-	import MainFooter from '../../lib/components/MainFooter.svelte';
+    import { visible } from '$lib/actions';
+    import { Main } from '$lib/layouts';
+    import { DEFAULT_DESCRIPTION, DEFAULT_HOST } from '$lib/utils/metadata';
+    import { TITLE_SUFFIX } from '$routes/titles';
+    import FooterNav from '../../lib/components/FooterNav.svelte';
+    import MainFooter from '../../lib/components/MainFooter.svelte';
 
-	const title = 'Assets' + TITLE_SUFFIX;
-	const description = DEFAULT_DESCRIPTION;
-	const ogImage = DEFAULT_HOST + '/images/open-graph/website.png';
+    const title = 'Assets' + TITLE_SUFFIX;
+    const description = DEFAULT_DESCRIPTION;
+    const ogImage = DEFAULT_HOST + '/images/open-graph/website.png';
+
+    enum Section {
+        NAMING = 'Naming',
+        LOGOTYPE = 'Logotype',
+        LOGOMARK = 'Logomark',
+        COLORS = 'Brand colors',
+        VISUALS = 'Product visuals',
+        CONTACT = 'Contact us'
+    }
+
+    const getSectionId = (section: Section) => section.toLowerCase().replace(/\s/g, '-');
```

Comment:
fyi, this part we probably wanna re-use for pages like `/privacy`

Reply:
Agreed! I was thinking if these pages should use markdoc though?